### DISCUSSION TITLE
Outpost manager doesn't want to give missions (after taking some other mission and save-loading)

### DISCUSSION THREAD

**Discussion by SomeRandomNoobKekeke** [FakeFishGames/Barotrauma#15145] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

So, i entered the outpost
I took "praise the honkmother" mission at the airlock and went to the doctor to heal the crew
And then i realized: "oh, it's that laggy outpost with the club, i finally found it", save-quited and went recording https://github.com/FakeFishGames/Barotrauma/discussions/15144
But after loading, outpost manager didn't have missions anymore, even though I didn't talk to him

### Reproduction steps

1. load [save](https://github.com/user-attachments/files/17740647/guh.zip) (don't mind the mods and mildly bitten crew)
2. take "praise the honkmother" mission
3. save-load 
4. manager doesn't have any missions now

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Rokvach** [comment]
Fixed and tested the fix to be working correctly, fix coming in the next update. Closing discussion.
